### PR TITLE
Generated OpenAPI client for Datacite MDS API

### PR DESCRIPTION
**Pull request by norrs**
It generates a project under the `$buildir/generated` .. 

But here is some example what the OpenAPI generator tool does.

### CODE REVIEW

**Comment by norrs** [comment]
Based on https://support.datacite.org/openapi/5d7a4aa4b0df7300e72ee074 , had to add responses to the openapi specification.

It might be an idea to actually have this as a standalone pipeline maybe, since the generator generates a project. And we probably want to build that project and publish its jar from there..

**Comment by axthosarouris** [comment]
Will this create an OpenApi specification for the endpoints of the project?

File: datacite-mds-client/openapi_datacite_mds.json [hunk 1]
```diff
@@ -0,0 +1,625 @@
+{
```
Comment (axthosarouris) [inline datacite-mds-client/openapi_datacite_mds.json:1]: where is this hosted?

**Review by axthosarouris — APPROVED** [review]

**Review by janbj — DISMISSED** [review]
This is okay for now.

**Comment by norrs** [comment]
This was just approved, merging. Only solved a simply merge conflict after additional modules were added to the project from another merge request.

**Pull request merged**